yt_dlp/extractor/microsoftembed.py

```diff
@@ -63,3 +71,221 @@ def _real_extract(self, url):
             'subtitles': subtitles,
             'thumbnails': thumbnails,
         }
+
+
+class MicrosoftMediusBaseIE(InfoExtractor):
+    @staticmethod
+    def _sub_to_dict(subtitle_list):
+        subtitles = {}
+        for sub in subtitle_list:
+            subtitles.setdefault(sub.pop('tag', None) or 'unknown', []).append(sub)
```

Comment:
I know the yt-dlp codestyle™ prefers `or` over setting the default value in `get`, but I think here for `pop` it's fine
```suggestion
            subtitles.setdefault(sub.pop('tag', 'unknown'), []).append(sub)
```